packages/venia-ui/lib/components/Header/cartTrigger.css

```diff
@@ -1,9 +1,29 @@
-.root {
+.root_desktop {
     composes: root from '../clickable.css';
     height: 3rem;
     min-width: 3rem;
 }
 
+.root_mobile {
+    composes: root_desktop;
+    display: none;
+}
+
+.background {
+    align-items: center;
+    display: grid;
```

Comment:
The desktop cart trigger now sits inside this `background` div. 

If the site header ever decides to align its items differently (to the bottom, for example), this code will have to be updated too otherwise the cart trigger will be out of sync.

If there's a better way to do this I'd be all for it.

Reply:
No longer relevant.